sites/fast-website/src/public/index.html

```diff
@@ -23,7 +23,7 @@
               <ul>
                 <li><fast-anchor appearance="lightweight" href="https://explore.fast.design">Components</fast-anchor></li>
                 <li><fast-anchor appearance="lightweight" href="#">Integration</fast-anchor></li>
-                <li><fast-anchor appearance="lightweight" href="https://www.fast.design/docs/en/contributing/install">Documentation</fast-anchor></li>
+                <li><fast-anchor appearance="lightweight" href="/docs/fast-foundation/getting-started">Documentation</fast-anchor></li>
```

Comment:
Should probably go to `/docs/introduction`

Reply:
Good catch Rob.  We'll want everything to be relative so that when they the same files move through the Front Door into staging environment or production the entire site is functional.